server/api/index.js

```diff
@@ -1,6 +1,63 @@
 const express = require("express");
 const router = express.Router();
+const db = require("../helpers/db");
 
 router.get("/status", (req, res) => res.send({ status: "OK" }));
 
+// get '/classes'
+router.get("/classes", (req, res) => {
+    db.getClasses().then(
+            data => {
+                res.send(data);
+            });
+});
+
+// get '/classes/:id'
```

Comment:
Comments like this aren't very helpful. You don't have to have a comment here. But if you use one, make it more descriptive, like:

```
// Define a route to return a single class object
```

The same is true for the other route's comments.